configs/getConfig.js

```diff
@@ -0,0 +1,20 @@
+const _ = require("lodash");
```

Comment:
also when using lodash in node code, it generally makes more sense to call the variable itself `lodash`, not `_`, since it's much for informative to grep for a full word string like `lodash` than it is to grep for a single character `_` (especially with Node's `___filename` and `__dirname` constants)